package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "node index.js",
-    "test": "./node_modules/.bin/standard && ./node_modules/.bin/ava",
+    "test": "standard && ava",
```

Comment:
I revert this change because in windows not recognized ./node_modules/... command, I will specify standard and ava global as requirements  in the Readme

Reply:
What about using `npx ava` ?